src/database/migrations/20191031103713-add-numberOfSpace-bookings.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = {
```

Comment:
Why do we need this new column?

Reply:
A user may need to book more than space like a room

Reply:
Is that not captured in the `Accommodations` model? [here](https://github.com/andela/eagle-bn-backend/blob/7d3ce2456674eb66cb95419d8a2b9fcbab2a08af/src/database/models/Accommodations.js#L72)

Reply:
No, availableSpace is justliek a number of rooms that hotel has, but numberOfSpace is a number of rooms a user will need to book